ecl/sss/v1/approval_requests/testing/fixtures.go

```diff
@@ -9,72 +9,101 @@ import (
 const idApprovalRequest1 = "9a76dca6-d8cd-4391-aac6f-2ea052f10f4"
 const idApprovalRequest2 = "fc578e8b-dea2-4f8c-aa7e-9026fa173632"
 
-const startTime = "2018-07-26 08:40:01"
+var actionsBody = []byte(
+	`{` +
+		`"firewall":` +
+		`{` +
+		`"availability_zone":"zone1-groupa",` +
+		`"default_gateway":"",` +
+		`"description":"abcdefghijklmnopqrstuvwxyz",` +
+		`"firewall_plan_id":"bd12784a-c66e-4f13-9f72-5143d64762b6",` +
+		`"name":"abcdefghijklmnopqrstuvwxyz",` +
+		`"tenant_id":"6a156ddf2ecd497ca786ff2da6df5aa8"` +
+		`}` +
+		`}`)
 
 var listResponse = fmt.Sprintf(`
 {
-  "approval_requests": [
-    {
-      "request_id": "%s",
-      "external_request_id": "test007",
-      "approver_type":"tenant_owner",
-      "approver_id":"11a98bf9cb144af5a204c9da566d2bd0",
-      "request_user_id":"ecid9999888881",
-      "service":"network",
-      "action" : [
-        {
-          "service": "network",
-          "region": "jp1",
-          "api_path": "/network/v1/firewall",
-          "method": "POST",
-          "body": "request's body json"
-        }
-      ],
-      "descriptions": [
-        {
-          "lang": "en",
-          "text": "approval resquest test"
-        }
-      ],
-      "request_user": false,
-      "approver": true,
-      "approval_deadline": "2017-02-05 09:45:22",
-      "approval_expire": null,
-      "registered_time": "2017-01-31 07:43:13",
-      "updated_time": null,
-      "status": "registered"
-    },
-    {
-      "request_id": "%s",
-      "external_request_id": "test006",
-      "approver_type":"tenant_owner",
-      "approver_id":"66a98bf9cb1238192a204c9da566dbd0",
-      "request_user_id":"ecid9999888882",
-      "service":"network",
-      "action" : [
-        {
-          "service": "network",
-          "region": "jp1",
-          "api_path": "/network/v1/firewall",
-          "method": "POST",
-          "body": "request's body json"
-        }
-      ],
-      "descriptions": [
-        {
-          "lang": "en",
-          "text": "approval resquest test"
-        }
-      ],
-      "request_user": false,
-      "approver": true,
-      "approval_deadline": "2016-12-25 09:45:22",
-      "approval_expire": null,
-      "registered_time": "2016-12-13 02:20:21",
-      "updated_time": null,
-      "status": "expired"
-    }
-  ]
+	"approval_requests": [
```

Comment:
Do not we need to test string body as well as json?

Reply:
I wrote test code with actions.body as a string.